lib/views/home/home_view_model.dart

```diff
@@ -17,8 +20,12 @@ class HomeViewModel extends StateNotifier<HomeState> {
   final StreamController<List<String>> _surveyListStream = StreamController();
 
   final ProfileUseCase _profileUseCase;
```

Comment:
It seems other use cases are missing the prefix as well 😅 our use case always has the verb to understand the right behavior of the use case FYI
![image](https://user-images.githubusercontent.com/60863885/219581354-74cb249a-01fa-4b0b-b8ce-7a8cfb37f145.png)